tools/quake3/q3map2/model.cpp

```diff
@@ -295,7 +295,17 @@ static std::map<ModelNameFrame, AssModel> s_assModels;
    loads a picoModel and returns a pointer to the picoModel_t struct or NULL if not found
  */
 
-static AssModel *LoadModel( const char *name, int frame ){
+static AssModel *LoadModel( const char *rawName, int frame ){
+	// attempt to canonicalize name across platforms
+	char name[1024] = {0};
```

Comment:
no need to add overflowable and basically copypasted code, there is this way of doing this operation: https://github.com/Garux/netradiant-custom/blob/master/tools/quake3/common/vfs.cpp#L302